src/Http/Http/ref/Microsoft.AspNetCore.Http.netcoreapp3.0.cs

```diff
@@ -17,6 +17,25 @@ public partial class ApplicationBuilder : Microsoft.AspNetCore.Builder.IApplicat
 }
 namespace Microsoft.AspNetCore.Http
 {
+    public partial class BindingAddress
+    {
+        public BindingAddress() { }
+        public string Host { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        public bool IsUnixPipe { get { throw null; } }
+        public string PathBase { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        public int Port { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        public string Scheme { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        public string UnixPipePath { get { throw null; } }
+        public override bool Equals(object obj) { throw null; }
+        public override int GetHashCode() { throw null; }
+        public static Microsoft.AspNetCore.Http.BindingAddress Parse(string address) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public static partial class BufferingHelper
```

Comment:
I think this should be removed and we should just add the useful method to one of the existing extensions classes.